akka-docs/src/main/paradox/stream/operators/StreamConverters/fromOutputStream.md

```diff
@@ -4,14 +4,34 @@ Create a sink that wraps an `OutputStream`.
 
 @ref[Additional Sink and Source converters](../index.md#additional-sink-and-source-converters)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Sink.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala) { #fromOutputStream }
-
-@@@
+@apidoc[StreamConverters.fromOutputStream](StreamConverters$) { scala="#fromOutputStream(out:()=%3Cjava.io.OutputStream,autoFlush:Boolean):akka.stream.scaladsl.Sink[akka.util.ByteString,scala.concurrent.Future[akka.stream.IOResult]]" java="#fromOutputStream(akka.japi.function.Creator)" }
 
 ## Description
 
-TODO: We would welcome help on contributing descriptions and examples, see: https://github.com/akka/akka/issues/25646
+Creates a Sink which writes incoming `ByteString`s to an `OutputStream` created by the given function.
+
+Materializes a @java[`CompletionStage`]@scala[`Future`] of `IOResult` that will be completed with the size of the file (in bytes) at the streams completion,
```

Comment:
```suggestion
Materializes a @java[`CompletionStage`]@scala[`Future`] of `IOResult` that will be completed with the size of the file (in bytes) on completion,
```

Reply:
I'm not 100% convinced about this change. And this writeing is used in multiple places (I've found the exact match in about 10 other locationsin the code). Maybe fix it in a separate PR?